.github/CODEOWNERS

```diff
@@ -40,11 +40,19 @@ doc_classes/*        @godotengine/documentation
 /modules/regex/      @LeeZH
 /modules/upnp/       @godotengine/network
 /modules/websocket/  @godotengine/network
+/modules/gdnative/text/ @bruvzg
+/modules/text_server_*/ @bruvzg
```

Comment:
Please keep alphabetical ordering.

Reply:
Fixed.

Reply:
Also changed `server` to `servers` in existing lines.